components/status/StatusCard.vue

```diff
@@ -95,12 +96,13 @@ const showReplyTo = $computed(() => !replyToMain && !directReply)
       <template v-if="status.inReplyToAccountId">
         <StatusReplyingTo
           v-if="showReplyTo"
-          ml-20px pt-1 pl-5
+          m="l-5" rtl-m="l-0 r-5"
+          p="t-1 l-5" rtl-p="l-0 r-5"
           :status="status"
           :is-self-reply="isSelfReply"
           :class="faded ? 'text-secondary-light' : ''"
         />
-        <div flex="~ col gap-1" items-center pos="absolute top-0 left-0" w="77px" z--1>
+        <div flex="~ col gap-1" items-center pos="absolute top-0 left-0" rtl="pos-left-auto right-0" w="77px" z--1>
```

Comment:
From contributing guide:

For absolute positioned elements, don't use `left/right`: for example `left-0`. Use `inset-inline-start/end` instead. `UnoCSS` shortcuts are `inset-is` for `inset-inline-start` and `inset-ie` for `inset-inline-end`. Example: `left-0` should be replaced with `inset-is-0`.